### PR TITLE
Convert `answer-corrector` to TypeScript

### PR DESCRIPTION
**Pull request by zinovik**
The issue: https://github.com/kantord/LibreLingo/issues/790

### CODE REVIEW

File: workspaces/answer-corrector/index.test.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'jest';
```
Comment (kantord) [inline workspaces/answer-corrector/index.test.ts:1]: hmm, is it necessary to import jest? I'm not saying it's incorrect, I've just never seen this before

**Comment by kantord** [comment]
Thanks for the PR!!

It looks almost perfect.

the CI seems to be failing for 2 reasons:

- [jest is failing](https://kantord.semaphoreci.com/jobs/a99e07c9-bdb2-4cd4-ad74-d41e8792e1df), I think something might be wrong with the TypeScript config there?
- the web build is failing, there is something weird going on:https://kantord.semaphoreci.com/jobs/ab37e40a-53a1-4984-951d-f324c624764e not sure why it's even trying to import the test file? It looks also like some kinf of a configuration issue

File: workspaces/answer-corrector/index.test.ts (hunk 1 above)
Comment (zinovik) [reply workspaces/answer-corrector/index.test.ts:1]: I have imported it to remove this error:
`Cannot find name 'describe'`.
But let's try another solution ;)

**Comment by kantord** [comment]
Hi @zinovik , how is it going? Do you need some help here?

**Comment by zinovik** [comment]
> Hi @zinovik , how is it going? Do you need some help here?

I've updated it after your previous comment. Doesn't it work now?

**Comment by kantord** [comment]
Hi @zinovik , you can see the failures if you click here:

2020-10-16-231854_970x616_scrot

**Comment by kantord** [comment]
Now everything is good, but jest is failing: apparently it doesn't recognize the typescript file

**Comment by zinovik** [comment]
> Now everything is good, but jest is failing: apparently it doesn't recognize the typescript file

Cool, I will fix it

**Review by kantord — APPROVED** [review]

**Comment by kantord** [comment]
Hi @zinovik ! Looks good to me now! :100: 

Thank you for your contribution!

@allcontributors please add @zinovik for code

**Comment by zinovik** [comment]
@kantord Thank you! But I am not sure it's finished. Tests don't fail now, but they are not run now. It's some work to do here.

```
yarn ts-jestPassed in
yarn run v1.22.5
$ /home/semaphore/LibreLingo/node_modules/.bin/ts-jest
​
Usage:
  ts-jest command [options] [...args]
​
Commands:
  config:init           Creates initial Jest configuration
  config:migrate        Migrates a given Jest configuration
  help [command]        Show this help, or help about a command
​
Example:
  ts-jest help config:migrate
Done in 0.08s.
```

**Comment by kantord** [comment]
Hi @zinovik ok!! Feel free to create a new PR to fix those issues.

**Comment by zinovik** [comment]
> Hi @zinovik ok!! Feel free to create a new PR to fix those issues.

@kantord No problem, it's done here: https://github.com/kantord/LibreLingo/pull/822

**Pull request merged**